.expeditor/verify_private.pipeline.yml

```diff
@@ -115,18 +115,7 @@ steps:
 
   - label: ":semgrep: Security"
     command:
-      - echo "running in $(pwd)"
-      - export SEMGREP_BRANCH=$BUILDKITE_BRANCH
-      - export SEMGREP_REPO_NAME=chef/automate
-      # Activates links to buildkite builds in slack notification
-      - export SEMGREP_JOB_URL=$BUILDKITE_BUILD_URL
-      # Activates links to git commits in slack notification
-      - export SEMGREP_REPO_URL=https://github.com/chef/automate
-      - export BASELINE=$BUILDKITE_PULL_REQUEST_BASE_BRANCH
-      - \[ "$BUILDKITE_BRANCH" != "master" \] && \[ -n "\$BASELINE" \] && echo "PR build on '$BUILDKITE_BRANCH' branch; base is '\$BASELINE'"
-      - \[ "$BUILDKITE_BRANCH" != "master" \] && \[ -z "\$BASELINE" \] && echo "manual build on '$BUILDKITE_BRANCH' branch; using master as base" && BASELINE=master
-      - \[ "$BUILDKITE_BRANCH" == "master" \] && echo "build on master; using 'master~1' as base branch" && BASELINE=master~1
-      - python -m semgrep_agent --publish-token "\$SEMGREP_TOKEN" --publish-deployment \$SEMGREP_ID --baseline-ref \$BASELINE
+      - .expeditor/buildkite/semgrep.sh "$BUILDKITE_BRANCH" "$BUILDKITE_BUILD_URL" "$BUILDKITE_PULL_REQUEST_BASE_BRANCH"
```

Comment:
Curious; looking at some other shell files in our code base, it seemed to be the case that these buildkite env variables should have been directly accessible in the script but they were not--I had to manually pass them in.

Reply:
You need to change `propagate-environment` to `propagate-environment` (with an `a`)